dashboard/public/pixelation/pixelation.js

```diff
@@ -37,6 +37,65 @@ function pixelationInit() {
     bitsPerPixelRange.setAttribute("disabled", "true");
     startOver.setAttribute("disabled", "true");
   }
+
+  customizeStyles();
+  initProjects();
+}
+
+function customizeStyles() {
+  if (!window.options) {
+    // Default is version 3 (all features enabled).
+    window.options = {version: '3'};
+  }
+  if (options.version == '1') {
+    $('.hide_on_v1').hide();
+  } else if (options.version == '2') {
+    $('.hide_on_v2').hide();
+    $('#height, #width').prop('readonly', true);
+  }
+  if (options.hex) {
+    $('input[name="binHex"][value="hex"]').prop('checked', true);
+  }
+  if (options.instructions) {
+    $('#below_viz_instructions').text(options.instructions).show();
+  }
+}
+
+function initProjects() {
+  // Initialize projects for save/load functionality if channel id is present.
+  if (appOptions.channel) {
+    window.apps.setupProjectsExternal();
+    var sourceHandler = {
+      setInitialLevelHtml: function (levelHtml) {},
+      getLevelHtml: function () {
+        return '';
+      },
+      setInitialLevelSource: function (levelSource) {
+        options.projectData = levelSource;
+      },
+      getLevelSource: function () {
+        return pixel_data.value.replace(/[ \n]/g, "");
+      }
+    };
```

Comment:
it could, but since it is just a parameter to project.init, I'm not sure I see a reason to move it out. Moving it out would make it possible to call the functions on it from elsewhere in this file, but I'm not sure we want to encourage that.